locations/locationController.spec.js

```diff
@@ -114,6 +115,7 @@ describe('POST/ add locations', () => {
         'description must be a string',
         'image_url must be a string',
         'address must be a string',
+        'place_id key is required, but value can be empty',
```

Comment:
likewise this on line 118 which will break if you change the iniitlal message. so it will need a change